src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -685,6 +686,16 @@ public class SimpleEvents {
 				.requiredPlugins("MC 1.16+");
 		}
 
+			Skript.registerEvent("Player Shear Entity", SimpleEvent.class, PlayerShearEntityEvent.class, "player (shear[s]|shearing) [an] entity")
+				.description("Called when a player shears an entity.")
+				.examples(
+					"on player shearing an entity:",
+						"\tsend \"You sheared %event-entity%\" to player",
+						"chance of 5%",
+						"repair event-slot"
+				)
+				.since("INSERT VERSION")
+				.requiredPlugins("MC 1.13+, MC 1.15+ (event-item, event-slot)");
```

Comment:
Mostly the majority of this class is double tabbed, I see it's been used improper for the paper events at the bottom.
```suggestion
					.description("Called when a player shears an entity.")
					.examples(
						"on player shearing an entity:",
							"\tsend \"You sheared %event-entity%\" to player",
							"chance of 5%",
							"repair event-slot"
					)
					.since("INSERT VERSION")
					.requiredPlugins("MC 1.13+, MC 1.15+ (event-item, event-slot)");
```

Reply:
I should remove "MC 1.13+" as future Skript drops legacy support.

Reply:
Do I keep it as is to be clear that this event exists in 1.13+ but the event-item and event-slot exists in 1.15+?

Reply:
While I don't believe the MC 1.13+ is necessary I believe it's best to add that as it explains and gives more information about this event

Reply:
Remving 1.13+ is okay, `MC 1.15+ (event-item, event-slot)` is clear enough